project_task_stage_allow_timesheet/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 ACSONE SA/NV
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Project Task Stage Allow Timesheet",
+    "summary": """
+        Allows to tell that a task stage is opened for timesheets.""",
+    "version": "18.0.1.0.1",
```

Comment:
```suggestion
    "version": "18.0.1.0.0",
```
Hey, looks like you forgot to reset the version :smiley: